### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Fireworkstars46** [Dhranios/Luigi-s-Mansion#47] [Q&A] [answered]
Is there any etc for the next version of the map?

**Comment by Dhranios** [comment, marked as answer]
No. I'm taking my time to get as much right as I can. I have a lot more on my to-do list.
That is also why the Milestone named "v1.4" has no due date.

**Comment by Dhranios** [comment]
The update released 3 days ago, and a fix update for some issues today @Fireworkstars46.
